tidy3d/components/data/sim_data.py

```diff
@@ -213,13 +213,103 @@ def at_centers(self, field_monitor_name: str) -> xr.Dataset:
 
         return monitor_data.colocate(**xyz_kwargs)
 
-    def get_intensity(self, field_monitor_name: str) -> xr.DataArray:
+    def get_poynting_vector(self, field_monitor_name: str) -> xr.Dataset:
+        """return `xarray.Dataset` of the Poynting vector at Yee cell centers.
+
+        Calculated values represent the instantaneous Poynting vector for time-domain fields and the
+        complex vector for frequency-domain: S = 1/2 E × conj(H).
+
+        Only the available components are returned, e.g., if the indicated monitor doesn't include
+        field component `"Ex"`, then `"Sy"` and `"Sz"` will not be calculated.
+
+        Parameters
+        ----------
+        field_monitor_name : str
+            Name of field monitor used in the original :class:`Simulation`.
+
+        Returns
+        -------
+        xarray.DataArray
+            DataArray containing the Poynting vector calculated at the center locations of the Yee
+            grid.
+        """
+
+        field_dataset = self.at_centers(field_monitor_name)
```

Comment:
Might be good to mention in the docstring that the Poynting vector components are computed based on colocated fields at cell centers?